tests/testStreamFramesProcessed.cpp

```diff
@@ -55,7 +55,7 @@ TEST_P(StreamFramesProcessed, VerifyFramesProcessed) {
 
     AudioStreamDataCallback *callback = new FramesProcessedCallback();
     mBuilder.setDirection(direction)
-            ->setFormat(AudioFormat::Float)
+            ->setFormat(AudioFormat::I16)
```

Comment:
Why was this changed to I16?

Reply:
OpenSL ES doesn't support Float below API 23

Reply:
For input streams, OpenSL ES doesn't support Float below API 23. For output streams, OpenSL ES doesn't support Float below API 21.